style.css

```diff
@@ -0,0 +1,832 @@
+}
+
+.card p,
+.card h3,
+.cardh p,
+.cardh h3 {
+  font-family: 'Lato', sans-serif;
+}
+
+.card h3,
+.cardh h3 {
+  color: #272a31;
+  font-weight: 900;
+}
+
+.card p,
+.cardh p {
+  font-size: small;
+}
+
+.button {
+  display: flex;
+  justify-content: center;
+}
+
+.btn2 {
+  background-color: white;
+  border: #d1d1d1 1px solid;
+  padding: 10px 45%;
+  margin-bottom: 30px;
+  background-image: url(Images/Icons/more-arrow.png);
+  background-repeat: no-repeat;
+  background-position: top 50% right 40%;
+  background-size: 5%;
+}
+
+.logo p {
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+  font-size: small;
+}
+
+.logo img {
+  width: 80%;
+  border: #d1d1d1 1px solid;
+  padding: 10px 10px;
+}
+
+.past {
+  padding-top: 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-bottom: 20px;
+  border-top: #d1d1d1 1px solid;
+  row-gap: 10px;
+}
+
+.past p {
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+}
+
+.past h3 {
+  text-align: center;
+  color: white;
+  font-size: xx-large;
+  font-family: 'CocoGoose', sans-serif;
+}
+
+.old1 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-image: url(Images/lbaa1.png);
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  box-shadow: inset 0 0 0 100px rgba(236, 82, 66, 0.74);
+  height: 200px;
+  color: white;
+}
+
+.old2 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-image: url(Images/lbaa2.jpg);
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  box-shadow: inset 0 0 0 100px rgba(236, 82, 66, 0.74);
+  height: 200px;
+  color: white;
+}
+
+.oldcont {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 80%;
+  row-gap: 15px;
+}
+
+footer {
+  display: flex;
+  flex-direction: column;
+  row-gap: 0;
+}
+
+.partners {
+  background-color: #272a31;
+  width: 100%;
+  height: 300px;
+  background-image: url(Images/Partners/partners.jpg);
+  background-repeat: no-repeat;
+  background-size: auto 100%;
+  background-position: top left 50%;
+}
+
+.copyrights {
+  padding-block: 5%;
+  background-color: white;
+  width: 100%;
+  height: 100px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.copyrights p {
+  font-family: 'Lato', sans-serif;
+}
+
+.topbar {
+  display: none;
+}
+
+.twitter {
+  background: url(Images/Icons/twiticon.png);
+  background-size: 100%;
+  background-repeat: no-repeat;
+  height: 20px;
+  width: 20px;
+  display: inline-block;
+}
+
+.facebook {
+  background: url(Images/Icons/faceicon.png);
+  background-size: 40%;
+  background-repeat: no-repeat;
+  height: 20px;
+  width: 20px;
+  display: inline-block;
+}
+
+.WP {
+  display: none;
+}
+
+@media only screen and  (min-width: 768px) {
+  .topbar {
+    display: inline-block;
+    background-color: #424242;
+    width: 100%;
+    padding-block: 7px;
+    text-align: end;
+  }
```

Comment:
- Let's ensure both headers  have the same padding right to match the design

*your design*
![headerr](https://user-images.githubusercontent.com/59397159/181863161-f41748be-286a-4b4f-b65a-283ce1c00dfa.png)
*main design*
![heaxx](https://user-images.githubusercontent.com/59397159/181863159-349aad27-187a-49c8-a567-343fd97d1fcd.png)

**index and about page**

Reply:
I added the padding requested and had to change both containers to flex to achieve it, it won't look exactly because of the lack of buttons or links but I think is very approached with my amount.